### PR TITLE
Timeout option

### PR DESCRIPTION
**Pull request by ramilexe**

### CODE REVIEW

File: cmd/watch.go [hunk 1]
```diff
@@ -88,6 +88,9 @@ func watch() {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 
+	fmt.Printf("Timeout is %d", timeout)
+	return
```
Comment (i-norden) [inline cmd/watch.go:92]: This return should be removed

File: cmd/watch.go [hunk 2]
```diff
@@ -88,6 +88,9 @@ func watch() {
 	ticker := time.NewTicker(5 * time.Second)
 	defer ticker.Stop()
 
+	fmt.Printf("Timeout is %d", timeout)
```
Comment (i-norden) [inline cmd/watch.go:91]: Should use `logWithCommand` to log messages if we want to keep this debug message

**Review by i-norden — CHANGES_REQUESTED** [review]

File: cmd/watch.go (hunk 1 above)
Comment (ramilexe) [reply cmd/watch.go]: yes, it was part of my local debug code. missed it. thank you

File: cmd/watch.go (hunk 2 above)
Comment (ramilexe) [reply cmd/watch.go]: fixed

**Review by i-norden — APPROVED** [review]

**Pull request merged**